packages/client/tests/functional/_utils/relationMode/computeSchemaHeader.ts

```diff
@@ -0,0 +1,41 @@
+import { match } from 'ts-pattern'
+
+import { Providers } from '../providers'
+
+export type ComputeSchemaHeader = {
+  provider: Providers
+  previewFeatures: string
+  relationMode: string
+}
+
+export function computeSchemaHeader({ provider, previewFeatures, relationMode }: ComputeSchemaHeader): string {
+  const USE_PLANETSCALE = false
```

Comment:
Preparation for later?

Reply:
Manual boolean, we need something else later